trenchboot/01-without-drtm.robot

```diff
@@ -0,0 +1,65 @@
+*** Settings ***
+Library             OperatingSystem
+Library             Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+Resource            ../sonoff-rest-api/sonoff-api.robot
+Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../variables.robot
+Resource            ../keywords.robot
+Resource            ../keys.robot
+Resource            ../lib/trenchboot.robot
+Resource            ../lib/tpm.robot
+
+Suite Setup         TrenchBoot Suite Setup
+Suite Teardown      Run Keyword
+...                     Log Out And Close Connection
+# WOD -- WithOut Drtm
+#
+# These tests verify sanity of the platform when DRTM is not employed and should
+# precede tests with DRTM on.
+#
+# Supported TPM: 1.2 or 2.0 (SHA1 and/or SHA256 PCR banks)
+
+
+*** Test Cases ***
+WOD001.001 All cores are up
+    [Documentation]    Verifies that all CPUs are online without DRTM because
+    ...    its easy to do and this could potentially catch an issue.
+    ${offline}=    Execute Command In Terminal
+    ...    cat /sys/devices/system/cpu/offline
+    Should Be Equal As Strings    ${offline}    ${EMPTY}
+    ...    Not all cores are up
+
+WOD002.001 Event log exists
+    [Documentation]    Event log should be present even without DRTM.
```

Comment:
DRTM log will be in different place.
```suggestion
WOD002.001 SRTM event log exists
    [Documentation]    SRTM event log should be present even without DRTM.
```

Reply:
https://github.com/Dasharo/open-source-firmware-validation/compare/5163e1a22995279403dcc7f277ea309cd4ac79b3..f3c14936b5b14706e5efa3208f4544fe2ee65cd3#diff-7e5ab2b2ca7ecfb07ecd58004916bea6a39f406b2ccb9c3cf7a9ffdd90218011R50